GOVERNANCE.md

```diff
@@ -0,0 +1,182 @@
+# Overview
+
+This is a meritocratic, consensus-based community project. Anyone with an
+interest in the project can join the community, contribute to the project
+design and participate in the decision making process. This document describes
+how that participation takes place and how to set about earning merit within
+the project community.
+
+# Roles and responsibilities
+
+## Users
+
+Users are community members who have a need for the project. They are the most
+important members of the community and without them the project would have no
+purpose. Anyone can be a user; there are no special requirements.
+
+The project asks its users to participate in the project and community as much
+as possible. User contributions enable the project team to ensure that they are
+satisfying the needs of those users. Common user contributions include (but are
+not limited to):
+
+- evangelising about the project (e.g. a link on a website and word-of-mouth
+awareness raising)
+- informing developers of strengths and weaknesses from a new user perspective
+- providing moral support (a ‘thank you’ goes a long way)
+
+Users who continue to engage with the project and its community will often
+become more and more involved. Such users may find themselves becoming
+contributors, as described in the next section.
+
+## Contributors
+
+Contributors are community members who contribute in concrete ways to the
```

Comment:
The user sections also talks about "user contributions" and it says `User contributions enable the project team to ensure that they are satisfying the needs of those users.` - which is bug reports.